server/src/main/java/org/apache/iotdb/db/metadata/template/Template.java

```diff
@@ -30,49 +39,70 @@
 
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 
+import java.util.ArrayDeque;
 import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Deque;
 import java.util.HashMap;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+
 public class Template {
   private String name;
 
-  private Map<String, IMeasurementSchema> schemaMap = new HashMap<>();
+  private IMNode mountedNode;
+  private Set<String> alignedPrefix;
+  private List<String> mountedPath;
+  private int measurementsCount;
+  private List<String> allSeriesPaths;
+  private Map<String, IMeasurementSchema> schemaMap;
```

Comment:
Is allSeriesPaths duplicated since it can be acquired by invoking schemaMap.keySet().

Reply:
removed